crates/wast/src/parser.rs

```diff
@@ -264,6 +264,16 @@ pub trait Peek {
         }
     }
 
+    /// The same as `peek2`, except it checks the token immediately following
+    /// the token immediately following the current token.
+    fn peek3(mut cursor: Cursor<'_>) -> bool {
+        if cursor.advance_token().is_some() {
+            Self::peek(cursor)
+        } else {
+            false
+        }
```

Comment:
Am I misreading this or is this function identical to `peek2`?

Reply:
Ah turns out this was dead code!